app/models/concerns/deliverable.rb

```diff
@@ -7,6 +7,10 @@ module Deliverable
   included do
     enum :delivery_status, (FAILED_DELIVERY_STATUS + DELIVERED_STATUS).index_by(&:itself)
     validates :last_brevo_webhook_received_at, presence: true, if: -> { delivery_status.present? }
+
+    scope :pending_or_delivered, lambda {
+      where.not(delivery_status: FAILED_DELIVERY_STATUS).or(where(delivery_status: nil))
```

Comment:
gros nitpick: je pense que ça revient au même mais plus en phase avec le nom du scope 
```suggestion
      where(delivery_status: DELIVERED_STATUS).or(where(delivery_status: nil))
```

Reply:
Ça revient au même effectivement mais je voulais anticiper le fait qu'il puisse y avoir d'autres cas qui puissent être non nil et non failed, mais en l'état en effet y'en a pas donc ok, ça sera plus clair